### PR TITLE
Prevent clickthrough when progress bar clicked during ad

### PR DESCRIPTION
**Pull request by incompl**
Based on #212

### CODE REVIEW

File: src/plugin.scss [hunk 1]
```diff
@@ -3,8 +3,12 @@ Styles for ad integrations.
 */
 
 // Prevent seeking during ad playback
-.vjs-ad-playing.vjs-ad-playing .vjs-progress-control {
-  pointer-events: none;
+.vjs-ad-playing.vjs-ad-playing .vjs-progress-control::after {
```
Comment (gkatsev) [inline src/plugin.scss]: Does sass transform the double colon (`::`) into a single colon (`:`)? IE8 only supports a single colon.

File: src/plugin.scss [hunk 2]
```diff
@@ -3,8 +3,12 @@ Styles for ad integrations.
...
+  pointer-events: auto;
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  content: "\000a0"
```
Comment (gkatsev) [inline src/plugin.scss]: Would it be better to just use a space?

File: src/plugin.scss (hunk 1 above)
Comment (gkatsev) [inline src/plugin.scss]: We should potentially disallow user selection on this element.

File: src/plugin.scss (hunk 2 above)
Comment (incompl) [inline src/plugin.scss]: I'm fine with space, just copied this from a previous version

File: src/plugin.scss (hunk 1 above)
Comment (incompl) [inline src/plugin.scss]: Nope it doesn't, good catch

**Comment by pagarwal123** [comment]
QA-LGTM

**Pull request merged**